gulpfile.js

```diff
@@ -8,11 +8,26 @@ var sourcemaps = require('gulp-sourcemaps');
 var webserver = require('gulp-webserver');
 var jshint = require('gulp-jshint');
 var watch = require('gulp-watch');
+var karma = require('karma').server;
 
 var getBundleName = function () {
   return 'sww';
 };
 
+gulp.task('tests', function () {
+  karma.start({
+    configFile: __dirname + '/testing/karma.conf.js',
+    singleRun: true
+  });
+});
+
+gulp.task('sw-tests', function () {
```

Comment:
Here is the problem I detected: I would keep sw and non sw tests seprated by now as sw environment is pretty unstable and I don't want it to interfere with the regular one. Furthermore, I discovered when running both of them, both runs are interfering. I think the best idea is to have both tasks separated, then make another task for all the tests by forcing the gulp orchestrator to run the environments one by one. Make sense?